go/api/base_client.go

```diff
@@ -3492,3 +3492,197 @@ func (client *baseClient) XClaimJustIdWithOptions(
 	}
 	return handleStringArrayResponse(result)
 }
+
+// Returns stream entries matching a given range of IDs.
+//
+// See [valkey.io] for details.
+//
+// Parameters:
+//
+//	key   - The key of the stream.
+//	start - The start position.
+//	        Use `options.NewStreamBoundary()` to specify a stream entry ID and its inclusive/exclusive status.
+//	        Use `options.NewInfiniteStreamBoundary()` to specify an infinite stream boundary.
+//	end   - The end position.
+//	        Use `options.NewStreamBoundary()` to specify a stream entry ID and its inclusive/exclusive status.
+//	        Use `options.NewInfiniteStreamBoundary()` to specify an infinite stream boundary.
+//
+// Return value:
+// A `map` of key to stream entry data, where entry data is an array of
```

Comment:
```suggestion
//  A `map` of key to stream entry data, where entry data is an array of
```
indentation missing.